### PR TITLE
makeMaxBatchSizePublic

### PR DESCRIPTION
**Pull request by xinlian12**
change maxBatchSize setter/getter to public

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkExecutionOptions.java [hunk 1]
```diff
@@ -127,11 +127,11 @@ CosmosBulkExecutionOptions setMaxMicroBatchPayloadSizeInBytes(int maxMicroBatchP
         return this;
     }
 
-    int getMaxMicroBatchSize() {
+    public int getMaxMicroBatchSize() {
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkExecutionOptions.java:130]: javadoc comments missing

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkExecutionOptions.java (hunk 1 above)
Comment (xinlian12) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkExecutionOptions.java:136]: added

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 ### 4.57.0-beta.1 (Unreleased)
 
 #### Features Added
+* Changed `setMaxMicroBatchSize` and `getMaxMicroBatchSize` in `CosmosBulkExecutionOptions` to public. See [PR 39335](https://github.com/Azure/azure-sdk-for-java/pull/39335)
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:6]:
```suggestion
* Added public APIs `setMaxMicroBatchSize` and `getMaxMicroBatchSize` in `CosmosBulkExecutionOptions` - See [PR 39335](https://github.com/Azure/azure-sdk-for-java/pull/39335)
```

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkExecutionOptions.java [hunk 3]
```diff
@@ -127,11 +127,24 @@ CosmosBulkExecutionOptions setMaxMicroBatchPayloadSizeInBytes(int maxMicroBatchP
         return this;
     }
 
-    int getMaxMicroBatchSize() {
+    /**
+     * The maximum batching size for bulk operations. Once queued docs exceed this values the micro
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkExecutionOptions.java:131]:
```suggestion
     * The maximum batch size for bulk operations. Once queued docs exceed this value, the micro
```

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkExecutionOptions.java [hunk 4]
```diff
@@ -127,11 +127,24 @@ CosmosBulkExecutionOptions setMaxMicroBatchPayloadSizeInBytes(int maxMicroBatchP
...
+     *
+     * @return the max micro batch size.
+     */
+    public int getMaxMicroBatchSize() {
         return maxMicroBatchSize;
     }
 
-    CosmosBulkExecutionOptions setMaxMicroBatchSize(int maxMicroBatchSize) {
+    /**
+     * The maximum batching size for bulk operations. Once queued docs exceed this values the micro
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosBulkExecutionOptions.java:141]:
```suggestion
     * The maximum batch size for bulk operations. Once queued docs exceed this value, the micro
```

**Review by kushagraThapar — APPROVED** [review]

**Pull request merged**